app/scripts/views/base.js

```diff
@@ -838,7 +838,7 @@ define(function (require, exports, module) {
      *
      * @method invokeBehavior
      * @param {Function} behavior
-     * @returns {variant} behavior's return value if behavior is a function,
+     * @returns {Variant} behavior's return value if behavior is a function,
```

Comment:
Again, I realise you're only capitalising what's already here but `Variant` reminds me of the COM type. Maybe that's fine but I find it a little bit jarring.